common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -47,6 +48,11 @@
     private static final String NATIVE_RESOURCE_HOME = "META-INF/native/";
     private static final File WORKDIR;
     private static final boolean DELETE_NATIVE_LIB_AFTER_LOADING;
+    private static final boolean TRY_TO_PATCH_SHADED_ID;
+
+    // Just use a-Z and numbers as valid ID bytes.
+    private static final byte[] UNIQUE_ID_BYTES =
+            "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ".getBytes(CharsetUtil.US_ASCII);
```

Comment:
You should probably just used a uuid.  Google internally has >1500 instances of this exact string when I searched our code.

Reply:
I mean does it really matter as we use a random to pick the bytes then

Reply:
@carl-mastrangelo running OS X? :grin:

Reply:
@carl-mastrangelo I will merge this one... We can do a followup if needed.